actionpack/lib/action_controller/metal/helpers.rb

```diff
@@ -12,6 +12,12 @@ module ActionController
   # In previous versions of \Rails the controller will include a helper which
   # matches the name of the controller, e.g., <tt>MyController</tt> will automatically
   # include <tt>MyHelper</tt>. To return old behavior set +config.action_controller.include_all_helpers+ to +false+.
+  # You can include the aforementioned in any environment file you wish (e.g. config/environments/production.rb) or alternatively
+  # you can include it in your config/application.rb file (for example):
```

Comment:
I don't think we need "(for example)" here; it's clear that the following code snippet is an example.

Reply:
ok will remove. 

Shall I squash the commits and resubmit?

Reply:
Yes please 👍